raiden/api/python.py

```diff
@@ -637,6 +637,59 @@ def get_network_events(
             to_block=to_block,
         ), key=lambda evt: evt.get('block_number'), reverse=True)
 
+    def get_channel_history_events(
+            self,
+            token_address: typing.TokenAddress,
+            partner_address: typing.Address = None,
+    ):
+
+        if not is_binary_address(token_address):
+            raise InvalidAddress(
+                'Expected binary address format for token in get_token_network_events',
+            )
+        if partner_address and not is_binary_address(partner_address):
+            raise InvalidAddress('Expected binary address format for partner in channel deposit')
+
+        raiden_events = self.raiden.wal.storage.get_events_by_block(
+            from_block=0,
+            to_block='latest',
+        )
+        events = []
+        if partner_address:
+            channel_state = views.get_channelstate_for(
+                views.state_from_raiden(self.raiden),
+                self.raiden.default_registry.address,
+                token_address,
+                partner_address,
+            )
+            if channel_state:
+                events = [
+                    (block_number, event)
+                    for block_number, event in raiden_events
+                    if isinstance(event, EVENTS_EXTERNALLY_VISIBLE) and
+                    event.channel_identifier == channel_state.identifier
+                ]
+            else:
+                events = []
+        else:
+            def is_part_of_token_network(event):
```

Comment:
Why was this function introduced? ATM there is only one token network registry, meaning the `payment_network_identifier` cannot change, also the `token_address` wont change, therefore you don't need to recompute `token_network_identifier` every time.

Please revert to the previous style, which was simpler and has the same end result.